lib/client.js

```diff
@@ -217,10 +217,15 @@ class Client extends EventEmitter {
       // TODO move this[kCallbacks] from being an array. The array allocation
       // is showing up in the flamegraph.
       const cb = this[kCallbacks].shift()
+      const request = this[kRequests].shift()
+      let skipBody
```

Comment:
maybe this part would be better written like this
```js
const skipBody = request.method === 'HEAD'
if (!skipBody) {
// assign lastBody
}
// logic...
return skipBody
```

Reply:
yes!